src/theme/MDXComponents.tsx

```diff
@@ -87,6 +88,33 @@ const ParallelBlocks: React.FC<React.PropsWithChildren> = ({ children }) => {
   );
 };
 
+const DocsLink: React.FC<React.PropsWithChildren<ComponentProps<typeof Link>>> = ({
```

Comment:
important change 3: wrap all links in the docs. If the link is to `console.prisma.io` check query params and add them.